src/Notifications.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from 'react';
+
+interface NotificationsProps {
+  removeTodoIsClicked: boolean,
+  setRemoveTodoIsClicked: React.Dispatch<React.SetStateAction<boolean>>,
+  addTodoIsClicked: boolean,
+  editTodoIsClicked: boolean,
+  setEditTodoIsClicked: React.Dispatch<React.SetStateAction<boolean>>,
+  setAddTodoIsClicked: React.Dispatch<React.SetStateAction<boolean>>,
+}
+
+export const Notifications: React.FC<NotificationsProps> = ({
+  removeTodoIsClicked,
+  setRemoveTodoIsClicked,
+  addTodoIsClicked,
+  editTodoIsClicked,
+  setEditTodoIsClicked,
+  setAddTodoIsClicked,
+}) => {
+  const [hideNotificationIsClicked,
```

Comment:
Consider renaming 'hideNotificationIsClicked' to a more descriptive name, such as 'isNotificationHidden'.